alexk_low_cost_robot/wizard/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "wizard"
```

Comment:
Could we name this something more specific such as `lcr-auto-configuration`, it might not be obvious what `wizard` is

Reply:
yeah you're right!